sources/web/datalab/static/util.js

```diff
@@ -1,7 +1,14 @@
 define(() => {
+
+  window.datalab = window.datalab || {};
+  window.datalab.debug = false;
```

Comment:
In general, I think it is OK to set the default false like this; in the case where a developer needs to see debug output during page load, he can edit the source code before reloading.
Another possibility would be to add a bit of code here to get location, grab the query parameters, and look for debug=true or something like that.